packages/components/forms/src/select/Select.styles.ts

```diff
@@ -0,0 +1,87 @@
+import tokens from '@contentful/f36-tokens';
+import { css, cx } from 'emotion';
+import { SelectWidth } from './types';
+
+function getStylesForSize(props: { width?: SelectWidth }) {
```

Comment:
I think we need to rethink the size prop. Selects should match inputs, and inputs need to match buttons in terms of height. 😁 But we can refactor later I guess.